ILSpy.ReadyToRun/ReadyToRunLanguage.cs

```diff
@@ -191,6 +196,7 @@ private class ReadyToRunReaderCacheEntry
 		{
 			public ReadyToRunReader readyToRunReader;
 			public string failureReason;
+			public Dictionary<EntityHandle, List<ReadyToRunMethod>> methodMap;
```

Comment:
Questions:

* Shouldn't a `Dictionary<EntityHandle, ReadToRunMethod>` be sufficient?
* Maybe we can switch to an array or `List<List<ReadyToRunMethod>>` if the "range" of `MethodDefinitonHandle`s that have (a) `ReadyToRunMethod`(s) is "densely populated" enough?